sbot/index.js

```diff
@@ -178,20 +178,6 @@ exports.init = function (ssb, config) {
     })
   )
 
-  // update ebt with latest block info
```

Comment:
just make sure you put this back when ebt gets enabled again 😱 😄

Reply:
Oops, replied [here](https://github.com/ssbc/patchwork/pulls#issuecomment-489331370) but I've made a note of it in [this issue](https://github.com/ssbc/patchwork/issues/1021). Thanks for pointing that out!